iron-collapse.html

```diff
@@ -142,7 +143,7 @@
     },
 
     updateSize: function(size, animated) {
-      this.enableTransition(animated);
+      this._updateTransition(animated);
```

Comment:
I thought of it, but I was concerned about the performance impact of adding a new property & it wasn't asked (yet).
Should I add it in this PR?